app/statistic_class.py

```diff
@@ -0,0 +1,58 @@
+import operator
+from typing import List, Optional
+
+from app.base import SharedSettings
+from app.dynamodb_dao import DynamoDBDao
+from app.models import PetStatistics
+
+
+class StatisticsSettings(SharedSettings):
+    s3_bucket_name: str
+    days: Optional[int]
+    email_title: str
+
+
+class Statistic:
+    def __init__(
+        self, dynamodb_table, s3_bucket, ses_service, settings: StatisticsSettings
+    ):
+        self.dynamodb_table = dynamodb_table
+        self.s3_bucket = s3_bucket
```

Comment:
currently you are passing s3_client, either pass s3_bucket or rename to s3_client